documentation/man/features/kw-deploy.rst

```diff
@@ -31,20 +31,17 @@ executed:
 
 You can specify the deploy target via command line by using the flag
 ``--remote <remote>:<port>`` (e.g., ``--remote 172.16.254.1:22``); however, if
-you do this frequently you will probably prefer to add this information to your
-local **kworkflow.config**. See the example below::
-
-  default_deploy_target=remote
-  ssh_user=root
-  ssh_ip=172.16.254.1
-  ssh_port=22
+you plan on deploying to the same target frequently you'll probably want to use
+``kw remote`` to setup and save your target's information to your local 
```

Comment:
How about something like:

You can benefit from the feature ``kw remote`` to save the ssh information in a configuration file used by kw; for more information, check ``kw remote --help``.

Also, I think you should keep the example but use the `kw remote`

Reply:
Both done.